### PR TITLE
resolves #{3}

### PR DESCRIPTION
**Pull request by wmh950419**
resolves #{3}

### CODE REVIEW

**Review by aplgit — APPROVED** [review]
Absolutely perfect......probably! :-D

**Review by SalehPouresmaeeli — APPROVED** [review]

File: divide.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```
Comment (t-rooker) [inline divide.py]:
```suggestion
def divide(x, y):
```

**Review by t-rooker — CHANGES_REQUESTED** [review]
move ```divide.py``` to ```pythoncalculator/divide.py```
move ```test_divide.py``` to ```tests/test_divide.py```
remove ```tests/__init__.py```

File: tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .divide import divide
```
Comment (bobturneruk) [inline tests/__init__.py:1]: This line may need to be removed.

Comment (bobturneruk) [reply tests/__init__.py:1]: As, from this location, Python probably can't find `divide` to import.

**Pull request closed without merging**